src/Directory.Build.props

```
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>Latest</LangVersion>
```

Use explicit version in case when the next language upgrade came and we haven't upgrade dotnet to match that language version
```suggestion
    <LangVersion>8.0</LangVersion>
```

---

why not use the latest?